standard_autoencoder.py

```diff
@@ -27,6 +27,10 @@ def __init__(self,
         self.learn_continuous_variance = learn_continuous_variance
 
     def init_network(self):
+        if self.learn_continuous_variance:
+            # we exponentiate this because it has to be non-negative. 
+            self.log_continuous_variance = tf.Variable(self.initialization_function([1]))
```

Comment:
I wonder if this should be on a per-phenotype basis?

Reply:
as discussed, this corresponds to shifting from PPCA to factor analysis, i think? maybe worth doing at some point, but will leave as is for now.